magpylib/_src/obj_classes/class_Collection.py

```diff
@@ -804,26 +773,27 @@ def getJ(
             `numpy.ndarray` object is returned. If 'dataframe' is chosen, a `pandas.DataFrame`
             object is returned (the Pandas library must be installed).
 
```

Comment:
Why removing this in this PR ??

Reply:
This was not removed but moved up. There was a docsting bug such that "output" was below "Returns".